CONTRIBUTING.md

```diff
@@ -4,6 +4,8 @@ This document is a guide to writing and editing documentation for the [Datadog D
 
 Some of these guidelines are enforced by [the Datadog docs implementation of the Vale linter][4]. After you make a PR, check its **Files changed** tab to see and fix warnings and errors flagged by the linter.
 
+See our [README][9] for additional contribution requirements, including the `<username>/<branch_name>` branch naming convention for all pull requests.
```

Comment:
```suggestion
See our [README][9] for additional contribution requirements, including the `<slack_username>/<branch_name>` branch naming convention for all pull requests.
```

Reply:
I want to point out that the Slack username is helpful for internal contributors, but external contributors might be confused about how to proceed.

Reply:
Never mind, i see this is under Datadog employees. Please ignore!